cellbrowser_tools/bin/build_release.py

```diff
@@ -2,22 +2,21 @@
 # -*- coding: utf-8 -*-
 
 import argparse
-from datetime import datetime
 import logging
-from pathlib import Path
 import traceback
+from datetime import datetime
+from pathlib import Path
 
 import dask
-from dask_jobqueue import SLURMCluster
-from distributed import LocalCluster
-from prefect import task, Flow, unmapped
-from prefect.engine.executors import DaskExecutor, LocalExecutor
+from aics_dask_utils import DistributedHandler
+from distributed import Client, LocalCluster
 
 # from fov_processing_pipeline import wrappers, utils
-from cellbrowser_tools import createJobsFromCSV
-from cellbrowser_tools import dataHandoffUtils
-from cellbrowser_tools import generateCellLineDef
-from cellbrowser_tools import validateProcessedImages
+from cellbrowser_tools import (createJobsFromCSV, dataHandoffUtils,
+                               generateCellLineDef, validateProcessedImages)
+from dask_jobqueue import SLURMCluster
+from prefect import Flow, task
+from prefect.engine.executors import DaskExecutor, LocalExecutor
```

Comment:
These are all auto sorted by my editor.

Reply:
That's fine; mostly I was just separating local imports from external ones

Reply:
Yea.... it _should_ have done that.... Don't know why it didn't. Normally it sorts by, `stdlib`, `external`, `internal`